CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Generated from:
```

Comment:
The hash sign is the syntax for the biggest heading - I do not think that is what you wanted here.

I have never used comments in markdown before, but you might find some valuable information at https://stackoverflow.com/questions/4823468/comments-in-markdown

Reply:
Thank you for bringing this up.
Fixed it in https://github.com/zopefoundation/ExtensionClass/pull/41/commits/de7422942142a175779e1217e32ce06b9b919009